manim/utils/tex_templates.py

```diff
@@ -20,6 +20,31 @@ def _new_ams_template():
     return TexTemplate(preamble=preamble)
 
 
+""" Tex Template preamble used by original upstream 3b1b """
+_3b3b_preamble = r"""
```

Comment:
Is there a reason why it is called _3b3b_preamble? I'd say it should be _3b1b_preamble

Reply:
ah crap.

Reply:
Welp. I'll push a fix directly to master.

Reply:
https://github.com/ManimCommunity/manim/commit/88bbdb156aa23c34b6eb07f0b4263d0d9deb01c3